examples/s2_field_combination_extern_rf_train/nrw_crop_extern_s2_workflow_to_datasets_with_timeshift.ipynb

```diff
@@ -0,0 +1,2364 @@
+       "0      1048.0      0  ...  1814.0        OE  1048.0   923.0  1380.0  1148.0   \n",
+       "1      1602.0      1  ...  3555.0        GT   670.0  1279.0  1992.0   604.0   \n",
+       "2      1492.0      2  ...  2369.0        GT   671.0   854.0  1883.0     NaN   \n",
+       "3       386.0      3  ...  1530.0        GL   465.0   440.0   238.0   378.0   \n",
+       "4       604.0      4  ...  3653.0        GL   699.0   726.0   418.0   523.0   \n",
+       "...       ...    ...  ...     ...       ...     ...     ...     ...     ...   \n",
+       "49419   916.0  49419  ...  2209.0        GT   552.0   649.0  1185.0   226.0   \n",
+       "49420   488.0  49420  ...  3954.0        GT  1264.0   598.0   395.0  1881.0   \n",
+       "49421   854.0  49421  ...  2379.0        GT   821.0   889.0  1062.0   483.0   \n",
+       "49422   858.0  49422  ...  4083.0        AF  1237.0   777.0   813.0  1057.0   \n",
+       "49423   619.0  49423  ...  4764.0        GL   668.0   833.0   419.0   284.0   \n",
+       "\n",
+       "        NDVI_-1                             geometry  \\\n",
+       "0      0.085486  MULTIPOINT (428690.027 5711938.189)   \n",
+       "1      0.134015  MULTIPOINT (427819.337 5710040.545)   \n",
+       "2      0.108832  MULTIPOINT (427320.866 5710158.178)   \n",
+       "3      0.397063  MULTIPOINT (431527.388 5693772.886)   \n",
+       "4      0.406943  MULTIPOINT (431535.193 5693614.690)   \n",
+       "...         ...                                  ...   \n",
+       "49419  0.146780  MULTIPOINT (472357.075 5696612.529)   \n",
+       "49420  0.449148  MULTIPOINT (472016.875 5697690.039)   \n",
+       "49421  0.200596  MULTIPOINT (471981.413 5696219.338)   \n",
+       "49422  0.429673  MULTIPOINT (471704.064 5697043.769)   \n",
+       "49423  0.268842  MULTIPOINT (472522.639 5708612.229)   \n",
+       "\n",
+       "                     time_start                  time_end  \n",
+       "0     2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "1     2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "2     2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "3     2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "4     2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "...                         ...                       ...  \n",
+       "49419 2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "49420 2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "49421 2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "49422 2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "49423 2021-10-01 00:00:00+00:00 2021-11-01 00:00:00+00:00  \n",
+       "\n",
+       "[49424 rows x 53 columns]"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gp_res = await workflow.vector_stream_into_geopandas(\n",
```

Comment:
I was just wondering… maybe we should show a plot in the beginning… or maybe (also) now?